Selu.383.SP24.Mobile/app.json

```diff
@@ -28,13 +28,15 @@
     "web": {
       "favicon": "./assets/favicon.png"
     },
- 
-    "owner": "kade.snyder",
+    "extra": {
+      "eas": {
+        "projectId": "ad6bd4e0-31ba-4621-b19f-3c6becb0f960"
+      }
+    },
+    "owner": "lillaundry",
```

Comment:
Once we merge this - please email me your expo token so I can set the secret for you since you don't have access in this repository